tests/mongodb/fixtures/ensg_by_symbol.json

```diff
@@ -0,0 +1,3 @@
+[{ "_id" : "BRCA1", "ensgs" : [ "ENSG00000012048" ] },
```

Comment:
Instead of committing a ton of test files can we either write these on the fly or provide StringIO objects to the functions that need them?

Making test files is usually unnecessary and makes unit testing hard to maintain because the test and the input data exist in two places instead of directly in the test suite.

Reply:
See here:

https://docs.python.org/3.6/library/io.html?highlight=stringio

Reply:
All future tests will be driven this way, but I wanted to give example files for using the import function I'd written for people who want to import data from file/db.